index.js

```diff
@@ -1,56 +1,44 @@
 /*
- * Main functions: core call infrastructure, letting setting up the room, event listeners, and joining
+ * Main functions: core call infrastructure, setting up the callframe and event listeners, creating room URL, and joining
  * Event listener callbacks: fired when specified Daily events execute
  * Call panel button functions: participant controls
  */
 
 /* Main functions */
+let callFrame, room;
 
-// Creates the callframe
-// Defines event listeners on Daily events
-// Assigns an event listener to the input field to change the join button color
-async function setup() {
-  callFrame = await window.DailyIframe.createFrame(
-    document.getElementById('callframe'),
-    {
-      iframeStyle: {
-        position: 'absolute',
-        top: '0',
-        left: '0',
-        width: '100%',
-        height: '90%',
-        border: '0',
-      },
-    }
-  );
+async function createCallframe() {
+  const callWrapper = document.getElementsByClassName('wrapper')[0];
+  callFrame = await window.DailyIframe.createFrame(callWrapper);
 
   callFrame
     .on('loaded', showEvent)
     .on('started-camera', showEvent)
     .on('camera-error', showEvent)
-    .on('joining-meeting', showEvent)
-    .on('joined-meeting', showCallDisplay)
-    .on('recording-started', showEvent)
-    .on('recording-stopped', resetRecordingButton)
-    .on('recording-stats', showEvent)
-    .on('recording-error', showEvent)
-    .on('app-message', showEvent)
-    .on('input-event', showEvent)
+    .on('joining-meeting', showLobby)
+    .on('joined-meeting', toggleCallPanel)
     .on('error', showEvent)
-    .on('participant-joined', updateParticipantInfoDisplay)
-    .on('participant-updated', updateParticipantInfoDisplay)
-    .on('participant-left', updateParticipantInfoDisplay)
-    .on('left-meeting', hideCallDisplay);
+    .on('left-meeting', toggleCallPanel);
 
-  let roomURL = document.getElementById('room-url');
-  const joinButton = document.getElementsByClassName('join-call')[0];
+  const roomURL = document.getElementById('url-input');
+  const joinButton = document.getElementsByClassName('start-button')[1];
+  const createButton = document.getElementsByClassName('start-button')[0];
```

Comment:
I think `getElementById` will be easier instead of all these `getElementsByClassName`